### ISSUE TITLE
Embed as wireframe ?

### ISSUE THREAD

**Issue by tucq88** [rxjs-playground/rxjs-playground.github.io#3]
Is it possible ? It's more easier to people to interact with playground right on their website/article rather than redirect to playground website.

**Comment by imbhargav5** [comment]
Definitely. It can be done. Would you like to attempt it?

**Comment by tucq88** [comment]
Tried to embed with Medium but seems no luck T_T I'll try with simpler approach. But I think we should support medium :P

**Comment by imbhargav5** [comment]
So perhaps we need to create a page which has a different layout that is easier to embed? It is embeddable but hard to work with. https://jsfiddle.net/17naowzm/

**Comment by tucq88** [comment]
Yeah, I think just need to work out with the stye for keeping things simple. I think we 
- need to detect if page is loaded inside a iframe  or not 
- if yes, load some extra appropriate styles that display better in an iframe 

https://stackoverflow.com/questions/326069/how-to-identify-if-a-webpage-is-being-loaded-inside-an-iframe-or-directly-into-t

Just some idea, maybe I'll try this weekend.

**Comment by imbhargav5** [comment]
Awesome. :D Thank you!! 💯
